### PR TITLE
update blis

### PR DESCRIPTION
**Pull request by svlandeg**
## Description
Allow higher pin of `blis` to match that of Thinc + few typo fixes + bump to  "3.0.0a34"

## Checklist
- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by adrianeboyd** [comment]
`setup.cfg` and `pyproject.toml` need to be modified, too. (I thought we had a test that caught when these were out-of-sync?)

I have an update to `setup.py` for `blis` that we may or may not want to publish as 0.7.2 before the final v3 is released that stops you with a warning before compiling if you have an older version of pip, since you rarely want to compile it yourself.

**Comment by svlandeg** [comment]
> `setup.cfg` and `pyproject.toml` need to be modified, too. (I thought we had a test that caught when these were out-of-sync?)
> 
hm, good point, why is that test not catching this? I'll have a look.

**Comment by svlandeg** [comment]
Ah, that test does fail locally. Whoops.

File: spacy/pipeline/morphologizer.pyx [hunk 1]
```diff
@@ -210,7 +210,7 @@ class Morphologizer(Tagger):
 
         examples (Iterable[Examples]): The batch of examples.
         scores: Scores representing the model's predictions.
-        RETUTNRS (Tuple[float, float]): The loss and the gradient.
```
[the remark this answers is not in the document]
Comment (svlandeg) [reply spacy/pipeline/morphologizer.pyx:213]: Alt Text

**Pull request merged**